### PR TITLE
Add start on certain runlevel for example configs

### PR DESCRIPTION
**Pull request by robinboening**

### CODE REVIEW

**Comment by seuros** [comment]
Thank you.

File: examples/upstart/manage-many/sidekiq.conf [hunk 1]
```diff
@@ -17,8 +17,9 @@
 
 description "Sidekiq Background Worker"
 
-# no "start on", we don't want to automatically start
-stop on (stopping sidekiq-manager or runlevel [06])
```
Comment (mperham) [inline examples/upstart/manage-many/sidekiq.conf]: You remove the sidekiq-manager event support.

Comment (seuros) [inline examples/upstart/manage-many/sidekiq.conf]: I restored it in master

**Comment by robinboening** [comment]
Oh, sorry. Thanks for catching.

File: examples/upstart/manage-many/sidekiq.conf (hunk 1 above)
Comment (robinboening) [inline examples/upstart/manage-many/sidekiq.conf]: Oh, sorry. Thanks for catching.

**Pull request merged**